app/js/components/cohorts/cohortModal.js

```diff
@@ -74,6 +75,10 @@ class ActionsComponent extends Component {
         }
     }
 
+    resetError(){
+        this.setState({error: null});
```

Comment:
Awesome resetError function @wanjikum @paulupendo , Do we have a reason why we are using `null` directly, I would like to suggest we use the empty datatype of `error` as opposed to using `null`. i.e if error is a string we setState to `''`, if it is an object or array we set it to `{}` or `[]` respectively. This is to aid code readability so another developer working on the project knows exactly what datatype `error` is as opposed to `null` which could mean anything.